### PR TITLE
[1LP][RFR] support for freezing on python3 and python2

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
after the other parts have been merged, this fixes tha travis issue and brings in quickstart for python3 support

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: requirements/template_docs.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ bottle
 bottle-sqlite
 cached-property
 deepdiff
-diaper
+diaper!=1.4.0
```
Comment (izapolsk) [inline requirements/template_docs.txt]: Q: what is the reason ?

**Comment by izapolsk** [comment]
@RonnyPfannschmidt, looks good to me but what are you going to do with broken travis  with python 3.6 and "RHEVMSystem is not availiable on python3" ?

**Comment by RonnyPfannschmidt** [comment]
@izapolsk any python3 errors are ignorable for now, this PR is about establishing the testing, not about making it pass

File: requirements/template_docs.txt (hunk 1 above)
Comment (RonnyPfannschmidt) [reply requirements/template_docs.txt]: that particular release is broken

**Pull request merged**